z/simd/search.go

```diff
@@ -1,4 +1,4 @@
-// +build 386 arm armbe arm64 mips mipsle mips64p32 mips64p32le ppc ppc64 ppc64le sparc
+// +build 386 arm armbe arm64 mips mipsle mips64p32 mips64p32le ppc ppc64 ppc64le sparc mips64 mips64le
```

Comment:
```suggestion
// +build !amd64
```

I think we don't need to enumerate all non-amd64 archs here.